Eatery Blue/UI/HomeScreen/HomeModelController.swift

```diff
@@ -78,7 +78,11 @@ class HomeModelController: HomeViewController {
             allEateries = eateries.filter { eatery in
                 return !eatery.name.isEmpty
             }.sorted(by: { lhs, rhs in
-                lhs.name < rhs.name
+                if (lhs.isOpen == rhs.isOpen){
```

Comment:
Remove the parentheses: `if lhs.isOpen == rhs.isOpen {`

Reply:
You can also use `$0` and `$1` instead of `lhs` and `rhs` in the trailing closure but doesn't really matter.